### PR TITLE
dumpling: add system database detail

### PR DESCRIPTION
**Pull request by pepezzzz**
add system database detail description.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add system database detail.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。更多 Dum
 
 #### 使用 `--where` 选项筛选数据
 
-默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
+默认情况下，Dumpling 会导出排除系统数据库（包含 mysql / sys / INFORMATION_SCHEMA / PERFORMANCE_SCHEMA / METRICS_SCHEMA / INSPECTION_SCHEMA）后整个 TiDB 集群内的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```
Comment (lichunzhu) [inline dumpling-overview.md:181]:
```suggestion
默认情况下，Dumpling 会导出排除系统数据库（包含 mysql / sys / INFORMATION_SCHEMA / PERFORMANCE_SCHEMA / METRICS_SCHEMA / INSPECTION_SCHEMA）后整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

File: dumpling-overview.md (hunk 1 above)
Comment (TomShawn) [inline dumpling-overview.md:181]:
```suggestion
默认情况下，Dumpling 会导出除系统数据库（包括 `mysql` 、`sys` 、`INFORMATION_SCHEMA` 、`PERFORMANCE_SCHEMA`、`METRICS_SCHEMA` 和 `INSPECTION_SCHEMA`）外整个 TiDB 集群内的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -178,7 +178,7 @@ Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。更多 Dum
 
 #### 使用 `--where` 选项筛选数据
 
-默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
+默认情况下，Dumpling 会导出排除系统数据库（包括 `mysql` 、`sys` 、`INFORMATION_SCHEMA` 、`PERFORMANCE_SCHEMA`、`METRICS_SCHEMA` 和 `INSPECTION_SCHEMA`）外整个 TiDB 集群内的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```
Comment (TomShawn) [inline dumpling-overview.md:181]:
```suggestion
默认情况下，Dumpling 会导出排除系统数据库（包括 `mysql` 、`sys` 、`INFORMATION_SCHEMA` 、`PERFORMANCE_SCHEMA`、`METRICS_SCHEMA` 和 `INSPECTION_SCHEMA`）外整个数据库内的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```

File: dumpling-overview.md [hunk 3]
```diff
@@ -178,7 +178,7 @@ Dumpling 同时还支持从 `~/.aws/credentials` 读取凭证文件。更多 Dum
 
 #### 使用 `--where` 选项筛选数据
 
-默认情况下，除了系统数据库中的表之外，Dumpling 会导出整个数据库的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
+默认情况下，Dumpling 会导出排除系统数据库（包括 `mysql` 、`sys` 、`INFORMATION_SCHEMA` 、`PERFORMANCE_SCHEMA`、`METRICS_SCHEMA` 和 `INSPECTION_SCHEMA`）外整个数据库内的表。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```
Comment (TomShawn) [inline dumpling-overview.md:181]:
```suggestion
默认情况下，Dumpling 会导出排除系统数据库（包括 `mysql` 、`sys` 、`INFORMATION_SCHEMA` 、`PERFORMANCE_SCHEMA`、`METRICS_SCHEMA` 和 `INSPECTION_SCHEMA`）外所有其他数据库。你可以使用 `--where <SQL where expression>` 来选定要导出的记录。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**